plugin-tailwindcss/src/mod.ts

```diff
@@ -1,19 +1,19 @@
-import type { TailwindPluginOptions } from "./types.ts";
 import { initTailwind } from "./compiler.ts";
 import type { FreshBuilder } from "fresh/dev";
 import type { App } from "fresh";
+import type { TailwindPluginOptions } from "./types.ts";
 
 export function tailwind<T>(
   builder: FreshBuilder,
   app: App<T>,
   options: TailwindPluginOptions = {},
 ): void {
-  let processor: ReturnType<typeof initTailwind> | null;
+  let processor: ReturnType<typeof initTailwind> | null = null;
```

Comment:
```suggestion
  const instance = await postcss(twPostcss({
    optimize: app.config.mode === "production",
    ...options,
  }));
```
Might this work? If so, we can remove lines 16-17 and `plugin-tailwindcss/src/compiler.ts`. It'd require `tailwind()`  to become async and be `await`ed where used, including documentation.

Reply:
I also realise we weren't passing `options` from `tailwind()` to `initTailwind()`. This snippet fixes that.

Reply:
thx so much